arm/README.md

```diff
@@ -0,0 +1,37 @@
+# GitHub Action for Managing Azure Resources
+
+This action can be used to create or update a resource group in Azure using the [Azure Resource Manager templates](https://azure.microsoft.com/en-in/documentation/articles/resource-group-template-deploy/)
+One can also use this delete a resource group, including all the resources within the resource group.
+
+
```

Comment:
Should we give information about using the azure-login action just like it is mentioned in azure-cli

Reply:
@dikhakha 
We have it already. 
```
action "Manage Azure Resources" {
  uses = "Azure/github-actions/arm@master"
  env = {
    AZURE_RESOURCE_GROUP = "<Resource Group Name"
    AZURE_TEMPLATE_LOCATION = "<URL or Relative path in your repository>"
    AZURE_TEMPLATE_PARAM_FILE = "<URL or Relative path in your repository>"
  }
  needs = ["Azure Login"]
}
```

Reply:
@dikhakha Are you suggesting something else ?

Reply:
To log into a Azure, we recommend using the [Azure Login](../azure-login) Action.

Reply:
Thnks Done